### PR TITLE
Issue litmuschaos#2103 added validation for Username

### PR DESCRIPTION
**Pull request by jabhishek87**
Improved the username input field of Create New User
section(Settings Page)
Update regex

Username now include Aplhanumeric Chars
with @. (at, dot)allowed as Special charecter

Signed-off-by: AbhishekJ <ia.malhotra@gmail.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [X ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ X] I have signed the commit for DCO to be passed.
- [X ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [X ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/utils/validate.ts [hunk 1]
```diff
@@ -19,6 +19,15 @@ export const validateEmail = (value: string) => {
   return false;
 };
 
+export const validateUsername = (value: string) => {
+  const usernameValid = /^[a-zA-Z\d@\.]+$/;
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/utils/validate.ts:23]:
```suggestion
  const usernameValid = /^[a-zA-Z\d-_@\.]+$/;
```
My username would not be validated with the regex currently present, this suggestion adds - and _ chars to the regex which is the apt validation for a username. Could you please make this suggested change!

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Comment by arkajyotiMukherjee** [comment]
This input field component is the process of getting scraped...but the validation is good.
@S-ayanide you can add this PR depending on the progress of litmus-ui integration with litmus-portal

**Comment by SarthakJain26** [comment]
Closing this PR as some of the users wants to keep the username field simple and don't want any validation for the same. We Will reconsider this if required in future :slightly_smiling_face:

**Comment by SarthakJain26** [comment]
Closing this PR as some of the users wants to keep the username field simple and don't want any validation for the same. We Will reconsider this if required in future slightly_smiling_face

**Pull request closed without merging**